src/org/gk/elv/SimpleInstanceTransferHandler.java

```diff
@@ -39,7 +39,8 @@
     public SimpleInstanceTransferHandler() {
```

Comment:
I don't think this class is related to what you are doing.

Reply:
I think you're right! Removed autoformatter config from Eclipse.